diff --git a/src/schema.rs b/src/schema.rs
index 9ca268e..3f2ae98 100644
--- a/src/schema.rs
+++ b/src/schema.rs
@@ -34,11 +34,36 @@ table! {
         token_amount -> Numeric,
         zil_amount -> Numeric,
         is_sending_zil -> Bool,
     }
 }
 
+table! {
+    pool_txs (id) {
+        id -> Uuid,
+        transaction_hash -> Varchar,
+        block_height -> Int4,
+        block_timestamp -> Timestamp,
+        initiator_address -> Varchar,
+        token_address -> Varchar,
+
+        token_amount -> Nullable<Numeric>,
+        zil_amount -> Nullable<Numeric>,
+
+        tx_type -> Varchar,
+
+        swap0_is_sending_zil -> Nullable<Bool>,
+
+        swap1_token_address -> Nullable<Varchar>,
+        swap1_token_amount -> Nullable<Numeric>,
+        swap1_zil_amount -> Nullable<Numeric>,
+        swap1_is_sending_zil -> Nullable<Bool>,
+
+        change_amount -> Nullable<Numeric>,
+    }
+}
+
 allow_tables_to_appear_in_same_query!(
     backfill_completions,
     chain_events,
     claims,
     distributions,
     liquidity_changes,
     swaps,
 );
